ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml

```diff
@@ -438,6 +438,31 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-upi
+- as: ibmcloud-ipi-private-fips-p2-f7
```

Comment:
Per https://github.com/openshift/release/pull/39410#issuecomment-1550985534, I think May was suggesting to update it to f14.

Reply:
> as suggesting to update it to f14.

Yes